README.md

```diff
@@ -65,6 +65,49 @@ As of v2.0, the storage solution for making this data available alongside the no
 
 Data relating to the **CSSP 20CRDS** tutorials is held online in an Azure Blob Storage Service. To access this data user will need a valid shared access signature (SAS) token.  The data is in [Zarr](https://zarr.readthedocs.io/en/stable/) format and the total volume is ~2TB. The data is in hourly, 3 hourly, 6 hourly, daily and monthly frequencies stored seperatrely under the `metoffice-20cr-ds` container on MS-Azure. Monthly data only is also via [Zenodo](https://zenodo.org/record/2558135).
 
+
+## AWS 
+
+# Create an EC2 instance
+
+* Click on Launch instance
+* Choose Amazon Linux 2 AMI (HVM) 64-bit machine, click select
+* Choose t2.2xlarge and click next: configure instance details
+* Choose subnet default eu-west-2c
+* In IAM role choose existing trainings-ec2-dev role and click next: storage
+* 8 gb is fine, click next: add tags
+* Add following tags
+  * Name: Instance name
```

Comment:
should this be set by the user to a unique name (and so in square brackets)

Reply:
changed